src/frontend/src/table.rs

```diff
@@ -425,7 +425,15 @@ impl DistTable {
                 table_name,
                 new_table_name,
             )
-            .await
+            .await?;
+            self.partition_manager
```

Comment:
Move this invocation into `move_table_route_value`? I think it's more reasonable, if you move or update the table route, you should invalidate the table route info.

Reply:
A little suggestion:  merge these two calls into a new function:

```rust
self.set_table_global_value(new_key, value).await?;
self.delete_table_global_value(key).await?;
```

Such as `self.update_table_global_value(key, new_key, value).await?;`

Reply:
Good suggestion, thx.😀